### PR TITLE
Add instructions to start Let's Chat

### PR DESCRIPTION
**Pull request by gbouquet**
Hey, I don't know if I missed something, but the instructions didn't seem to tell how to run the app. I figured it was `npm start` looking at `package.json`, but it's not on the home page. So I hope this pullrequest improves things! Aweomse project :)

### CODE REVIEW

File: _layouts/index.html [hunk 1]
```diff
@@ -102,6 +102,11 @@ <h3 class="lc-section-subheading">In just a few, simple steps...</h3>
 <pre>
 cp settings.yml.sample settings.yml
 </pre>
+                            <p>Run Let's Chat:
```
Comment (hhaidar) [inline _layouts/index.html]: Paragraph should terminate here!

**Comment by hhaidar** [comment]
Sweet, thanks! Just one small thing..

**Comment by gbouquet** [comment]
Hope this last commit fixes it! Are the instructions okay to you though?

**Comment by hhaidar** [comment]
@gbouquet yup, thanks!

**Comment by gbouquet** [comment]
Cool! This seems like an awesome project to contribute to :)

**Pull request merged**